OWML.ModHelper.Menus/BaseConfigMenu.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Newtonsoft.Json.Linq;
+using OWML.Common;
+using OWML.Common.Menus;
+using OWML.ModHelper.Events;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace OWML.ModHelper.Menus
+{
+    public abstract class BaseConfigMenu : ModPopupMenu, IBaseConfigMenu
+    {
+        public IModManifest Manifest { get; }
+
+        private readonly IModConsole _console;
```

Comment:
please adopt changes I made in my PR (that is make console `protected` in ModMenu)

Reply:
Kinda weird why you made it private in a first place. Almost every tier had it's own `_console`

Reply:
Totally agree

Reply:
I won't adopt those same changes in this PR tho, they'll be merged together to dev.